signinwithapplebutton/src/main/res/layout/sign_in_with_apple_button.xml

```diff
@@ -1,42 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="wrap_content"
-    android:layout_height="wrap_content"
-    android:layout_gravity="center"
-    android:duplicateParentState="true"
-    android:minHeight="@dimen/sign_in_with_apple_button_minimumHeight"
-    android:paddingStart="@dimen/sign_in_with_apple_button_horizontalPadding"
-    android:paddingEnd="@dimen/sign_in_with_apple_button_horizontalPadding">
```

Comment:
Do we want to include paddingStart and paddingEnd in this view too?

Reply:
I currently have the padding implemented in where I utilize the buttons. But if you think it would be better suited here. I can definitely do that!

Reply:
👍 From what I have seen from the usage, I think we would want centre the icon and the text. Now It's left aligned. Also, the Apple icon seems a bit too small.

I am ok with setting padding when you create the button. Up to you.